### PR TITLE
[12.0] run  copier on 12.0 branch to fix CI

### PR DESCRIPTION
**Pull request by robinkeunen**
Travis is broken on new 12.0 branches. Running copier makes this branch consistent with the newer ones.

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
Thanks

**Review by MiquelRForgeFlow — APPROVED** [review]

File: .copier-answers.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+convert_readme_fragments_to_markdown: false
+generate_requirements_txt: true
+github_check_license: true
+github_ci_extra_env: {}
+github_enable_codecov: false
+github_enable_makepot: true
+github_enable_stale_action: true
+github_enforce_dev_status_compatibility: false
+include_wkhtmltopdf: false
+odoo_test_flavor: OCB
```
Comment (MiquelRForgeFlow) [inline .copier-answers.yml]:
```suggestion
odoo_test_flavor: Both
```

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

File: .copier-answers.yml (hunk 1 above)
Comment (robinkeunen) [reply .copier-answers.yml]: I tried to save the planet, but not today. It's fixed :-)

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**